configs/antv.json

```
@@ -38,44 +38,31 @@
       ]
     },
     {
-      "url": "https://antv.alipay.com/zh",
+      "url": "https://antv.vision/zh.*",
```

```suggestion
      "url": "https://antv.vision/zh",
```
Let's use URL instead of regex. This start_url will have the same having as using the regex